coalib/coala_main.py

```diff
@@ -160,7 +160,7 @@ def run_coala(console_printer=None,
             if not autoapply:
                 section['default_actions'] = ''
             elif force_show_patch:
-                section['default_actions'] = '*: ShowPatchAction'
+                section['default_actions'] = '**: ShowPatchAction'
```

Comment:
Don't change this in this issue, I am still discussing how to write a good test for this (for ApplyPatchAction) on my PR. Open another issue for this (as done in https://github.com/coala/coala/issues/4957 ) and then change it as this issue is limited to changing the glob in .coafile.

Reply:
BTW you sure this problem also occurs for ShowPatchAction?